typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -17,4 +17,6 @@ export enum WarpRouteIds {
   EthereumZircuitPZETH = 'PZETH/ethereum-zircuit',
   InevmInjectiveINJ = 'INJ/inevm-injective',
   MantapacificNeutronTIA = 'TIA/mantapacific-neutron',
+  StrideEclipseTIA = 'TIA/stride-eclipse',
```

Comment:
nit: I think the chains need to be alphabetically sorted, at least thats how the warp ID generation logic works, we should try to be consistent

Reply:
yup it's `<TICKER>/alphabet-chain-order`